Jenkinsfile

```diff
@@ -0,0 +1,77 @@
+pipeline {
+  agent {
+    docker {
+      image 'ubuntu'
+      reuseNode false
+      args '-u root'
+    }
+    
+  }
+  stages {
+    stage('Pull down ChefDK') {
+      steps {
+        sh '''apt-get update
+apt-get install -y curl sudo git build-essential
+curl -L https://chef.io/chef/install.sh | sudo bash -s -- -P chefdk -c current'''
+      }
+    }
+    stage('Bundle') {
+      steps {
+        sh '''chef exec bundle install
+chef gem install chefstyle activesupport'''
+      }
+    }
+    stage('Checks') {
+      steps {
+        parallel(
```

Comment:
move this down to line 31? so the directory gets removed first then the three are run at the same time?